### PR TITLE
Hostprocess container test updates

### PR DESCRIPTION
**Pull request by marosset**
Signed-off-by: Mark Rossetti <marosset@microsoft.com>

#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:

This PR updates host-process containers tests to skip tests that are not compatible with changes being made in containerd v1.7

Related to updates added with https://github.com/kubernetes/enhancements/pull/3311

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

/sig windows
/triage accepted
/area test

### CODE REVIEW

File: test/e2e/windows/utils.go [hunk 1]
```diff
@@ -44,3 +49,20 @@ func waitForDeployment(getDeploymentFunc func() (*appsv1.Deployment, error), int
...
 	})
 }
+
+// gets the container runtime and version for a node
+func getNodeContainerRuntimeAndVersion(n v1.Node) (string, semver.Version, error) {
+	containerRuntimeVersionString := n.Status.NodeInfo.DeepCopy().ContainerRuntimeVersion
+	parts := strings.Split(containerRuntimeVersionString, "://")
+
+	if len(parts) != 2 {
+		return "", semver.Version{}, fmt.Errorf("Could not get container runtime and version from '%s'", containerRuntimeVersionString)
```
Comment (dcantah) [inline test/e2e/windows/utils.go:59]: Idioms usually to start errors in lowercase

**Review by dcantah — COMMENTED** [review]
small nit

File: test/e2e/windows/utils.go (hunk 1 above)
Comment (marosset) [reply test/e2e/windows/utils.go]: updated

**Pull request merged**